cmd/kaspawallet/daemon/server/create_unsigned_transaction.go

```diff
@@ -0,0 +1,96 @@
+package server
+
+import (
+	"context"
+	"github.com/kaspanet/kaspad/cmd/kaspawallet/daemon/pb"
+	"github.com/kaspanet/kaspad/cmd/kaspawallet/libkaspawallet"
+	"github.com/kaspanet/kaspad/util"
+	"github.com/pkg/errors"
+)
+
+func (s *server) CreateUnsignedTransaction(_ context.Context, request *pb.CreateUnsignedTransactionRequest) (*pb.CreateUnsignedTransactionResponse, error) {
+	s.lock.Lock()
+	defer s.lock.Unlock()
+
+	if !s.isSynced() {
+		return nil, errors.New("server is not synced")
+	}
+
+	err := s.refreshExistingUTXOs()
+	if err != nil {
+		return nil, err
+	}
+
+	toAddress, err := util.DecodeAddress(request.Address, s.params.Prefix)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: Implement a better fee estimation mechanism
+	const feePerInput = 1000
+	selectedUTXOs, changeSompi, err := s.selectUTXOs(request.Amount, feePerInput)
+	if err != nil {
+		return nil, err
+	}
+
+	changeAddress, err := s.changeAddress()
+	if err != nil {
+		return nil, err
+	}
+
+	psTx, err := libkaspawallet.CreateUnsignedTransaction(s.keysFile.ExtendedPublicKeys,
```

Comment:
Please search all over the code for "psTx" and rename to something that everyone understand.